### DISCUSSION TITLE
scrum prep Milestone 1 - 1

### DISCUSSION THREAD

**Discussion by Yahid1** [uprm-inso4116-2024-2025-s1/semester-project-trolley-tracker-app#16] [General]
Write your scrum here.

**Comment by MrL9000** [comment]
The past week has been a little all over the place, but overall it has been focused on getting everyone ready to start work on the project. As for me, I helped the manager test the fork from the RUMBus team that we will use to work on the project. After this, me, the manager, and a couple of others were shown a presentation from the RUMBus team explaning what the app is all about and second hand information of their efforts to get the project done. After that, I met up with the managers and teamleaders past Tuesday to tie up the teams, what will be done, etc. and, about an hour from when this message has been posted, I will meet with my team [the Team 1 (Metrics) team] to further organize the work we'll be doing in the coming weeks.

EDIT: After the meeting, I self assigned myself "[Lecture Topic Task] 1.2.1 current situation" mid meeting with Angel Martinez and the others from Team 1. Also, in the past 2 meetings, because we can't really code yet because, even if we do have a fork becauseit's an incomplete fork, we're currently focused on Doccumentation tasks and getting the ropes of Angular and the codebases used in the project for the time being.

**Reply by juan-ortiz92** [reply]
I have been setting up my depelopment area and catching up with information because of an issue with my github account.

**Comment by sebasagir** [comment]
was able to run the project via docker and via yarn start. 
<img width="2240" alt="Rumbus first run" src="https://github.com/user-attachments/assets/9ccba041-465a-4e75-bb49-9eb5f243ce5a">

Created issue for starting the terminology section of the documentation.

**Comment by Janiel777** [comment]
I've already started working on the documentation:
. 1.2.3 ideas
. 1.3.1 scope and span

I've already run the code we have from the RUMBus team and I'm learning how to use Angular.
![image](https://github.com/user-attachments/assets/265e54f8-bb5a-45b4-8689-f4b526d634a6)

I also followed up with my group so that they were doing what was recommended for this week.
![image](https://github.com/user-attachments/assets/1faa71d9-601b-469d-9647-c5cab4b25730)

**Comment by AngelMartinez33** [comment]
Have been helping mostly with administrative stuff, such as teams organization.
I was able to run the code, after downloading npm and yarn.
Had a reunion with blue team to make sure everybody understands what they're supposed to be doing now.
Focused on learning Angular.js and writing documentation.

**Comment by IvanierB** [comment]
This week I joined all team chats and attended team meetings to see the status of the project. Currently I am looking to select a task to help with the documentation since we are waiting for the rumBus team to give us access to the current source code so we can start developing.

**Comment by diegorobles2** [comment]
This week i've been having everything ready and set up on my workspace and been learning Angular.js.

**Comment by marcosarraiza** [comment]
This week I've been a bit busy overall but I communicated with all my teammates/managers and am looking to create/pick a task today to start working on it. Also setting up my warkspace to work with Angular alongside RUMBus.

**Comment by Yahid1** [comment]
This week I created some tasks, ensured the forked version of the Dash Board , created documentation on how to run the project, clarified milestone objectives, determined meeting schedule and presented the first meeting, assigned some tasks, clarified point system, assigned team leaders, set up project board for the issues, clarified team responsibilities, ensured the attendance of all members,, reviewed the system designed by RUMBus team. Currently working on implementing the system designed by the Professor.

**Comment by Brandell13** [comment]
This week I was preparing everything and setting it up for when it starts to be more effective and learning a little about angular.js

**Comment by dylanDeJesus21** [comment]
This week I attended team meeting and connected to all channels. I looked over the repository and did some research on angular. Also refreshed topics on Javascript

**Comment by dAninAd1248** [comment]
This week I've been learning angular and reading lecture presentations to get ideas for the documentation.

**Comment by JConcepcion01** [comment]
For this week I got started on setting up the project in my computer so that I could run everything and getting the base idea of what we're working on and what we're gonna do for this milestone

**Comment by yamayrie** [comment]
This past week I've been working on setting up and running the project. I've also been learning angular to be able to understand the code better.

**Comment by JoelTorres20** [comment]
This week I've been setting up the workspace amd little by little learning angular.js  and getting ready to contribute to the documentation.

**Comment by AlexisTraversoPR** [comment]
This week has been about learning and getting familiarized with the goals, objectives and vision of the project so I have not written any code yet. I attended team meetings and started to look into angular.js which will be used for the project

**Comment by rafa-quiles** [comment]
This week I've been setting up my workspace and getting the project running to familiarize myself with what's already there to see what I can work on.

**Comment by warm007** [comment]
During this week, I attended meetings and joined to the team chats, to get an update on the project's status. I set up my environment by installing and configuring everything. After cloning the GitHub repo and installing dependencies; I started learning AngularJS, focusing on key concepts like modules and controllers. While we wait for the RUM Bus team to provide access to the current source code, I'm selecting a task to help with documentation. Once the code is available, I’ll be ready to start working toward the first milestone.
